firmware/tunerstudio/rusefi.input

```diff
@@ -7,15 +7,20 @@
 ; this should stop TS from looking for the CAN ID in the 2nd byte location and allow the page reads to work correctly.
 enable2ndByteCanID = false
 
-#unset tuneByMAF
+;#unset tuneByMAF
 
-;[SettingGroups]
+[SettingGroups]
```

Comment:
did we intend to un-comment [SettingGroups]? why?

Reply:
We need [SettingGroups] because we're adding more options to it - see "settingGroup = tuneVeMode". We use the new option "tuneByTPS" to switch between table definitions (see all "#if tuneByTPS" below in the code)